plugins/scanners/fossid/src/main/kotlin/events/CloneRepositoryHandler.kt

```diff
@@ -208,3 +211,20 @@ class CloneRepositoryHandler(val config: FossIdConfig, val service: FossIdServic
         logger.info { "Data download has been completed." }
     }
 }
+
+// Deduplicates and normalizes ignore rules by removing duplicate directory and file rules,
+// and by stripping the trailing '/*' from directory rule values.
```

Comment:
If you're adding a comment to this function anyway, please turn in into proper function docs (inline KDoc format) with imperative mood wording.